solutions/fully-configurable/variables.tf

```diff
@@ -7,21 +7,39 @@ variable "ibmcloud_api_key" {
   description = "The IBM Cloud API key to deploy resources."
   sensitive   = true
 }
```

Comment:
skip_mysql_kms_auth_policy => check description in elasticsearch for `skip_elasticsearch_kms_auth_policy` and use it here